### ISSUE TITLE
doc: contains corrupt UTF-8; thus gem does not install cleanly

### ISSUE THREAD

**Issue by megahall** [gjtorikian/commonmarker#40]
```
$ sudo gem install commonmarker
Fetching: i18n-0.8.1.gem (100%)
Successfully installed i18n-0.8.1
Fetching: ruby-enum-0.7.1.gem (100%)
Successfully installed ruby-enum-0.7.1
Fetching: commonmarker-0.14.14.gem (100%)
Building native extensions.  This could take a while...
Successfully installed commonmarker-0.14.14
Parsing documentation for i18n-0.8.1
Installing ri documentation for i18n-0.8.1
Parsing documentation for ruby-enum-0.7.1
Installing ri documentation for ruby-enum-0.7.1
Parsing documentation for commonmarker-0.14.14
Installing ri documentation for commonmarker-0.14.14
ERROR:  While executing gem ... (ArgumentError)
    invalid byte sequence in UTF-8
$ echo $?
1
```

I'm happy to provide more data to debug this, if someone can explain how to get more verbose debug output. Right now the output is... more or less useless.

**Comment by megahall** [comment]
(ugly) workaround:

`sudo gem install --no-document commonmarker`

**Comment by kivikakk** [comment]
Thanks for this report! :Heart: I've just released v0.14.15, which fixes this bug.
